x-pack/plugins/apm/public/tutorial/tutorial_fleet_instructions/index.tsx

```diff
@@ -36,9 +38,15 @@ const CentralizedContainer = styled.div`
 
 type APIResponseType = APIReturnType<'GET /internal/apm/fleet/has_data'>;
 
-function TutorialFleetInstructions({ http, basePath, isDarkTheme }: Props) {
+function TutorialFleetInstructions({
+  http,
+  basePath,
+  isDarkTheme,
+  kibanaVersion,
+}: Props) {
   const [data, setData] = useState<APIResponseType | undefined>();
   const [isLoading, setIsLoading] = useState(false);
+  const isPrerelease = new SemVer(kibanaVersion).prerelease.length > 0;
```

Comment:
Maybe you could create a utils or helper function that will receive the kibana version and return if it is prerelease or not, and use it here and in the other files where the same logic is needed.